app/views/contacts/showLinkedIn.html.erb

```diff
@@ -69,9 +69,9 @@
     </div>
 
   </div>
-  <div class="row mapRow">
-    <div class="col removeLeftPadding">
-      <div id="contactMap">
+  <div class="row">
+    <div class="col">
+      <div id="contact-map">
 
       </div>
     </div>
```

Comment:
Wasn't strictly necessary, but removed all refs to custom classes to keep it clean